extensions/known extensions.md

```diff
@@ -0,0 +1,10 @@
+# Known Extension Documents for Homie
```

Comment:
Originally I thought that we include such a list into the main spec. But might indeed be a good idea to just link to this document instead. And the website can fetch this markdown file and render it somewhere additionally.